src/views/List.jsx

```diff
@@ -17,38 +18,59 @@ export function List({ data }) {
 		setSearchedItem('');
 	}
 
+	const navigate = useNavigate();
+
+	function handleNavigate(e) {
+		navigate('/add-item');
+	}
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
-			<form>
-				<label htmlFor="filter">Filter Items:</label>
-				<input
-					id="filter"
-					type="text"
-					inputmode="search"
-					value={searchedItem}
-					onChange={handleChange}
-					placeholder="Start Typing here..."
-				/>
-				<button
-					type="button"
-					onClick={buttonClick}
-					disabled={searchedItem.length === 0}
-				>
-					Clear
-				</button>
-			</form>
-			<ul>
-				{!filteredItems.length ? (
-					<p>It's not here!</p>
-				) : (
-					filteredItems.map((list) => {
-						return <ListItem name={list.name} key={list.id} />;
-					})
-				)}
-			</ul>
+			{/* welcome people to add to their list if it's empty */}
+			{data.length === 0 ? (
+				<div>
+					<h1>Hello!</h1>
+					<h2>
+						You have nothing on your list yet! Click here to add your first
+						item:
+					</h2>
+					<button onClick={handleNavigate} type="button">
```

Comment:
This button should probably be replaced by an actual link for accessibility purposes. A button is general meant to perform an action within a page, whereas the link is sending you to another page. Check out `<NavLink />` from `react-router-dom`.